dina-test-support/src/test/resources/missingRelation.json

```diff
@@ -17,6 +17,13 @@
       "description":{
         "en":"obj",
         "fr":"obj-fr"
+      },
+      "customObject": {
+        "name": {
+          "firstName": "firstName",
+          "lastName": "lastName"
+        },
+        "id": 1
```

Comment:
Again this test is designed to catch a missing relationship. Not missing attributes.